modules/deployment/src/main/java/org/mule/runtime/module/deployment/internal/RetryScheduledTask.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.deployment.internal;
+
+/**
+ * A retry scheduled task to execute an action.
+ */
+public interface RetryScheduledTask extends Runnable {
```

Comment:
where is this performAction invoked? It is basically the same as ActionTask, right?